### ISSUE TITLE
Write code to filter calendar and display

### ISSUE THREAD

**Issue by KthProg** [ITCS2250/HorseFarm#23]

**Comment by mgargano4** [comment]
I talked to schleiss after class and he emailed the lady we're doing this project for so maybe we can actually finish it. He cc'd the both of us but I want sure if your email was right lol. Anyways my email is mgargano4@Gmail.com, I figure we've gone this far we might as well try and finish

-----Original Message-----

**Comment by KthProg** [comment]
Yeah it's really close to completion, so I'm definitely willing to get the
rest of it completed. Plus it'll look nice on our GitHub accounts lol.
